components/dropindicator/index.css

```diff
@@ -10,54 +10,95 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+@media (forced-colors: active) {
+  .spectrum-DropIndicator {
+      --highcontrast-dropindicator-color: Highlight;
+  }
+}
+
+.spectrum-DropIndicator {
+  --spectrum-dropindicator-circle-border-color: var(--highcontrast-dropindicator-color, var(--spectrum-dropindicator-color));
+  --spectrum-dropindicator-border-color: var(--highcontrast-dropindicator-color, var(--spectrum-dropindicator-color));
```

Comment:
I don't have concerns about this not working, but generally we've been following a pattern of the high contrast and mods being with in the declarations below and not here where we're defining custom variables. 

So instead of:
`--spectrum-dropindicator-circle-border-color: var(--highcontrast-dropindicator-color, var(--spectrum-dropindicator-color));`

Using this(line 50): 
`border-color: var(--highcontrasts-dropindicator-circle-border-color, var(--mod-dropindicator-circle-border-color, var(--spectrum-dropindicator-circle-border-color)));`

Same concept for `--spectrum-dropindicator-border-color`

Reply:
I see, makes sense. Thanks for the feedback! Updated!